src/node_nanomsg.cc

```diff
@@ -23,6 +23,11 @@ using v8::Object;
 using v8::String;
 using v8::Value;
 
+#define ret NanReturnValue
+#define utf8 v8::String::Utf8Value
```

Comment:
It's a safe bet if compiling for io.js, since V8 requires C++11. But not safe in current node.js. the requirement for C++11 compatible compiler is why we didn't upgrade past 3.28.